pkg/kubelet/dockertools/manager.go

```diff
@@ -1497,7 +1499,7 @@ func (dm *DockerManager) createPodInfraContainer(pod *api.Pod) (kubeletTypes.Doc
 		return "", err
 	}
 
-	id, err := dm.runContainerInPod(pod, container, netNamespace, "")
+	id, err := dm.runContainerInPod(pod, container, netNamespace, "", "")
```

Comment:
It could but I don't think it's strictly required.  Since the ipc mode wasn't being passed I did the same.